src/fidgets/community/nouns-dao/NounishGovernance.tsx

```diff
@@ -22,6 +27,12 @@ export const nounishGovernanceConfig: FidgetProperties = {
       required: true,
       inputSelector: TextInput,
     },
+    {
+      fieldName: "daoContractAddress",
+      default: "Only for Builder Daos",
+      required: true,
```

Comment:
Is this actually required, or only necessary if other things are true? You don't need to fix this right now, but I am gauging how much conditional validation should be built soon

Reply:
its only necessary for builder daos, the eth main Nouns dao does not need that field. I wanted to get the structure of the components done so I can now improve the UX. Glad that worked though , just moon from here!